closure/goog/events/browserevent_test.js

```diff
@@ -137,6 +166,32 @@ function createBrowserEvent(type, button, opt_ctrlKey) {
   });
 }
 
+function createTargetTouchEvent(type, target, clientCoords, screenCoords) {
```

Comment:
This method is exactly the same as createChangedTouchEvent below. Can you consolidate into one?